antora.yml

```diff
@@ -12,7 +12,8 @@ asciidoc:
     tdcdnurl: https://cdn.tiny.cloud/1/_your_api_key_/tinydrive/6/tinydrive.min.js
     tinymce_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/6/tinymce.min.js
     tinydrive_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinydrive/6/tinydrive.min.js
-    webcomponent_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@1/dist/tinymce-webcomponent.min.js
+    webcomponent_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@2/dist/tinymce-webcomponent.min.js
+    jquery_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@2/dist/tinymce-jquery.min.js
```

Comment:
You appear to have branched this from `staging/docs-6` so it has things that shouldn't be released yet I guess. So 2 solutions:

1. Make this target `staging/docs-6` instead of `release/docs-6`, then we talk to @metricjs about doing a content release from staging.
2. Rebase this branch off `release/docs-6` instead (e.g. via `git rebase origin/release/docs-6 && git push -f`)

Reply:
I went with 2. above. It appeared to work but I got a curious message returned along the way.

`% git rebase origin/release/docs-6 && git push -f`
`Current branch hotfix/DOC-1669 is up to date.`
`Total 0 (delta 0), reused 0 (delta 0)`
`remote: `
`remote: GitHub found 2 vulnerabilities on tinymce/tinymce-docs's default branch (1 high, 1 moderate).`
`remote: To find out more, visit:`
`remote: https://github.com/tinymce/tinymce-docs/security/dependabot`
`remote: To https://github.com/tinymce/tinymce-docs.git`
` + 5d0dc186...df776123 hotfix/DOC-1669 -> hotfix/DOC-1669 (forced update)`

Reply:
Also, I don’t know how I got the branching wrong. I thought I was branching from `/release/docs-6`. Clearly, I wasn’t, but I managed to think I was. Sorry about that.

Reply:
Sigh sorry my suggestion didn't work because it just carried across all the other commits it had locally. I'll fix it instead, but basically I'm just going to add `-i` to the rebase command and manually drop the commits we don't want.

As for the 2 vulnerabilities we'll need to look into them. However, they are in dev dependencies so aren't as critical.